mod/groups/views/default/group/format/invitationrequest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * Group view for an invitation request
+ *
+ * @uses $vars['entity'] Group entity
+ */
+
+$user = elgg_get_logged_in_user_entity();
+$group = elgg_extract('entity', $vars);
+
+if (!$group instanceof \ElggGroup) {
+	return true;
+}
+
+$icon = elgg_view_entity_icon($group, 'small');
+
+$accept_url = elgg_http_add_url_query_elements('action/groups/join', array(
+	'user_guid' => $user->guid,
+	'group_guid'=> $group->guid,
+	));
+
+$accept_button = elgg_view('output/url', array(
+	'href' => $accept_url,
+	'is_action' => true,
+	'text' => elgg_echo('accept'),
+	'class' => 'elgg-button elgg-button-submit',
+	'is_trusted' => true,
+		));
+
+$delete_url = elgg_http_add_url_query_elements('action/groups/killinvitation', array(
+	'user_guid' => $user->guid,
+	'group_guid' => $group->guid,
+));
+
+$delete_button = elgg_view('output/url', array(
+	'href' => $delete_url,
+	'is_action' => true,
+	'confirm' => elgg_echo('groups:invite:remove:check'),
+	'text' => elgg_echo('delete'),
+	'class' => 'elgg-button elgg-button-delete mlm',
+		));
+
+$buttonbank = $accept_button . $delete_button;
```

Comment:
maybe not now.... but what if i want to add a "resend invitation" or something like that... if you do it like this (https://github.com/Elgg/Elgg/blob/22d14acba615693564a1c5cf485e60e47fe69280/mod/site_notifications/views/default/object/site_notification.php#L24), it is easily possible